src/main.rs

```diff
@@ -140,6 +140,43 @@ fn install_hook() -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+/// Copy the text te the clipboard.
+///
+/// This function never returns on Linux and various BSDs. See [here][1] for more info.
+///
+/// [1]: https://docs.rs/arboard/latest/arboard/trait.SetExtLinux.html#tymethod.wait
+fn copy_to_clipboard(s: String) -> Result<(), Box<dyn Error>> {
+    #[cfg(any(
+        target_os = "dragonfly",
+        target_os = "freebsd",
+        target_os = "illumos",
+        target_os = "linux",
+        target_os = "netbsd",
+        target_os = "openbsd",
+        target_os = "solaris"
+    ))]
```

Comment:
perhaps simpler, if we check for unix and !macos?

Reply:
No, because `nix::unistd::daemon` is only available for

```
#[cfg(any(target_os = "android",
          target_os = "dragonfly",
          target_os = "freebsd",
          target_os = "illumos",
          target_os = "linux",
          target_os = "netbsd",
          target_os = "openbsd",
          target_os = "solaris"))]
```

which is more restrictive than your proposal.

Reply:
Not sure why we need to cater for more OSes than those but ok, sure.

Reply:
> Not sure why we need to cater

Otherwise it wouldn't compile on some other unixes...